.storybook/grid/GridPanel.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { AddonPanel } from '@storybook/components';
+import { useChannel } from '@storybook/api';
+import { styles } from './gridStyles';
+import { INITIAL_IMAGE_EVENT, IMAGE_SELECTED_EVENT } from './withGrid';
+import { STORY_CHANGED } from '@storybook/core-events';
+import { IframePostMessageService } from '@guardian/grid-client';
+import { Reporter } from '@guardian/grid-client/lib/utils';
+
+const GRID_URL =
+    process.env.NODE_ENV === 'development'
+        ? 'https://media.test.dev-gutools.co.uk'
```

Comment:
I feel like these configuration constants are a bit scattered at the moment. Not urgent for now, but we might consider colocating these definitions in the future so that:

* it's easy to see what has been configured in one place
* it's simpler to keep these configurations in sync
* the around these bits of config (specifically how we detect dev mode) is not duplicated

Reply:
Agreed with all this 👍  As a first pass a single method to check whether we're in dev mode seems like a good place to start.

Reply:
I've done a little refactoring around this. I'm sure we can improve things further but I think I might save that for a future PR.